scim.sgnl.yaml

```diff
@@ -0,0 +1,419 @@
+      clientSecret: "{{Input Required: Client-Secret}}"
+      tokenUrl: "{{Input Required: AuthTokenURL}}"
+      scope: "{{Input Required: Scope}}"
+      authStyle: AutoDetect
+entities:
+  User:
+    displayName: SCIMUser
+    externalId: Users
+    description: User entity in SCIM
+    syncFrequency: HOURLY
+    syncMinInterval: 1
+    apiCallFrequency: SECONDLY
+    apiCallMinInterval: 1
+    pageSize: 100
+    pagesOrderedById: false
+    attributes:
+      - name: id
+        externalId: id
+        type: String
+        indexed: true
+        uniqueId: true
+      - name: externalId
+        externalId: externalId
+        type: String
+      - name: userName
+        externalId: userName
+        type: String
+      - name: formattedName
+        externalId: $.name.formatted
+        type: String
+      - name: familyName
+        externalId: $.name.familyName
+        type: String
+      - name: givenName
+        externalId: $.name.givenName
+        type: String
+      - name: middleName
+        externalId: $.name.middleName
+        type: String
+      - name: honorificPrefix
+        externalId: $.name.honorificPrefix
+        type: String
+      - name: honorificSuffix 
+        externalId: $.name.honorificSuffix
+        type: String
+      - name: displayName
+        externalId: displayName
+        type: String
+      - name: nickName
+        externalId: nickName
+        type: String
+      - name: profileUrl
+        externalId: profileUrl    
+        type: String
+      - name: userType
+        externalId: userType
+        type: String  
+      - name: title
+        externalId: title
+        type: String
+      - name: preferredLanguage
+        externalId: preferredLanguage
+        type: String
+      - name: locale  
+        externalId: locale
+        type: String
+      - name: timezone
+        externalId: timezone
+        type: String
+      - name: active
+        externalId: active
+        type: Bool 
+      - name: password
+        externalId: password
+        type: String
+      
+      # Enterprise User Extension
+      - name: employeeNumber
+        externalId: $..["urn:ietf:params:scim:schemas:extension:enterprise:2.0:User"].employeeNumber
+        type: String
+      - name: costCenter
+        externalId: $..["urn:ietf:params:scim:schemas:extension:enterprise:2.0:User"].costCenter
+        type: String
+      - name: organization
+        externalId: $..["urn:ietf:params:scim:schemas:extension:enterprise:2.0:User"].organization
+        type: String
+      - name: division
+        externalId: $..["urn:ietf:params:scim:schemas:extension:enterprise:2.0:User"].division
+        type: String
+      - name: department
+        externalId: $..["urn:ietf:params:scim:schemas:extension:enterprise:2.0:User"].department
+        type: String
+      - name: managerId
+        externalId: $..["urn:ietf:params:scim:schemas:extension:enterprise:2.0:User"].manager.value
+        type: String  
```

Comment:
The requirement is to create a "MANAGER" relationship between two SCIM Users so that snippet creators can perform "is X a manager of Y" checks. The id of the manager is extracted via a JSON path and ingested as `managerId`, an attribute entity of Entity `User`.

Can you please recommend the correct way to create relationships here.
cc: @rlenglet, @nholbrook